src/content/docs/en/pages/agreements/tos/terms-of-service.mdx

```diff
@@ -24,7 +24,7 @@ All Azion products and services are offered as Edge Services and available throu
+**Edge Caching:** an Edge Application's module designed to deliver web contents of all formats and types, including HTTP Live Streaming (HLS), via the HTTP/1.0, HTTP/1.1, HTTP/2 and HTTP/3 protocols, along with support for the HTTP GET, HEAD and OPTIONS methods. Includes access to (1) the Azion network for content delivery; (2) content caching feature with minimum TTL of 60 seconds; (3) HTTP header manipulation; (4) creation of redirecting or content delivery rules; (5) the Real-Time Purge tool; (6) delivery of cached content while the origin is inaccessible; (7) delivery of HTTPS (TLS) with OCSP Stapling support; (8) digital certificate uploads and support for the TLS x509 SNI format, in case it is supported by the user’s browser, or use of the shared certificate in Azion’s domain; (9) integration with Azion Products and Services. To use this service, Customers need to store the original version of their Content in an origin web server with HTTP access. THERE ARE NO GUARANTEES THAT CONTENTS (OBJECTS) WILL BE AVAILABLE IN CACHE THROUGHOUT THE ENTIRE TIME-TO-LIVE (TTL) CONFIGURED BY THE CUSTOMER. AZION MAKES THE NECESSARY INVESTMENTS TO KEEP DATA CACHED FOR THE CLOSEST TIME TO THAT CONFIGURED BY CUSTOMERS, HOWEVER, LEAST ACCESSED OBJECTS MAY BE AUTOMATICALLY DELETED FROM THE CACHE TO MAKE ROOM FOR MORE OFTEN ACCESSED OBJECTS VIA AN LRU (LEAST RECENTLY USED) ALGORITHM. MAXIMUM OBJECT SIZE IS 10GB. TO USE THE SERVICE WITH LARGER OBJECTS, CUSTOMERS ARE REQUIRED TO HAVE A WRITTEN AUTHORIZATION FROM AZION. Except as otherwise provided for in the Customer Agreement or any other documents attached thereto, Edge Caching will be billed according to the (a) Total Data Transferred; (b) Requests; and (c) Real-Time Purge Operations, according to the volumes consumed.
```

Comment:
```suggestion
**Edge Caching:** an Edge Application's module designed to deliver web contents of all formats and types, including HTTP Live Streaming (HLS), via the HTTP/1.0, HTTP/1.1, HTTP/2, and HTTP/3 protocols, along with support for the HTTP GET, HEAD and OPTIONS methods. Includes access to (1) the Azion network for content delivery; (2) content caching feature with minimum TTL of 60 seconds; (3) HTTP header manipulation; (4) creation of redirecting or content delivery rules; (5) the Real-Time Purge tool; (6) delivery of cached content while the origin is inaccessible; (7) delivery of HTTPS (TLS) with OCSP Stapling support; (8) digital certificate uploads and support for the TLS x509 SNI format, in case it is supported by the user’s browser, or use of the shared certificate in Azion’s domain; (9) integration with Azion Products and Services. To use this service, Customers need to store the original version of their Content in an origin web server with HTTP access. THERE ARE NO GUARANTEES THAT CONTENTS (OBJECTS) WILL BE AVAILABLE IN CACHE THROUGHOUT THE ENTIRE TIME-TO-LIVE (TTL) CONFIGURED BY THE CUSTOMER. AZION MAKES THE NECESSARY INVESTMENTS TO KEEP DATA CACHED FOR THE CLOSEST TIME TO THAT CONFIGURED BY CUSTOMERS, HOWEVER, LEAST ACCESSED OBJECTS MAY BE AUTOMATICALLY DELETED FROM THE CACHE TO MAKE ROOM FOR MORE OFTEN ACCESSED OBJECTS VIA AN LRU (LEAST RECENTLY USED) ALGORITHM. MAXIMUM OBJECT SIZE IS 10GB. TO USE THE SERVICE WITH LARGER OBJECTS, CUSTOMERS ARE REQUIRED TO HAVE A WRITTEN AUTHORIZATION FROM AZION. Except as otherwise provided for in the Customer Agreement or any other documents attached thereto, Edge Caching will be billed according to the (a) Total Data Transferred; (b) Requests; and (c) Real-Time Purge Operations, according to the volumes consumed.
```